server/api/audit_boards.py

```diff
@@ -274,18 +274,28 @@ def calculate_risk_measurements(election: Election, round: Round):
         round_contest.is_complete = is_complete
 
 
-def count_audited_votes(election: Election, round: Round):
-    vote_counts = dict(
-        BallotInterpretation.query.filter_by(is_overvote=False)
-        .join(SampledBallot)
-        .join(SampledBallotDraw)
-        .filter_by(round_id=round.id)
-        .join(BallotInterpretation.selected_choices)
-        .group_by(ContestChoice.id)
-        .values(ContestChoice.id, func.count())
-    )
+def count_audited_votes(round: Round):
+    for round_contest in round.round_contests:
+        contest = round_contest.contest
+        vote_counts = dict(
+            BallotInterpretation.query.filter_by(
+                contest_id=contest.id, is_overvote=False
+            )
+            .join(SampledBallot)
+            .join(SampledBallotDraw)
+            .filter_by(round_id=round.id)
+            .join(BallotInterpretation.selected_choices)
+            .group_by(ContestChoice.id)
+            .values(
+                ContestChoice.id,
+                func.count(
+                    SampledBallot.id
+                    if contest.is_targeted
+                    else SampledBallot.id.distinct()
```

Comment:
you've checked the SQL coming out of this and it looks good?

Reply:
I think so - not quite sure what I should look for but the query looks straightforward in each case.